### ISSUE TITLE
Fix warnings with Coq v8.13.x

### ISSUE THREAD

**Issue by khieta** [inQWIRE/SQIR#22]
`make all` succeeds for Coq v8.12.x and v8.13.x, but for 8.13 it spits out the following warning every time we add something to a Hint database... we should fix this before it becomes deprecated.
```
Warning: The default value for hint locality is currently "local" in a
section and "global" otherwise, but is scheduled to change in a future
release. For the time being, adding hints outside of sections without
specifying an explicit locality is therefore deprecated. It is recommended to
use "export" whenever possible. [deprecated-hint-without-locality,deprecated]
```

**Comment by k4rtik** [comment]
I was just fixing this issue in another Coq development. See https://coq.discourse.group/t/change-of-default-locality-for-hint-commands-in-coq-8-13/1140

**Comment by khieta** [comment]
Fixed with 7609b7d43795472bf34d31d244eef4c3bff8e9cf. Thanks for the tip @k4rtik!

**Comment by k4rtik** [comment]
Hi @khieta, I am not sure if it's fixed in all files, I am seeing a similar warning (sharing only one instance) in the branch for my PR #39, even though it includes your linked commit above:
```
File "./SQIR/UnitaryOps.v", line 402, characters 0-74:
Warning: The default value for rewriting hint locality is currently "local"
in a section and "global" otherwise, but is scheduled to change in a future
release. For the time being, adding rewriting hints outside of sections
without specifying an explicit locality attribute is therefore deprecated. It
is recommended to use "export" whenever possible. Use the attributes
#[local], #[global] and #[export] depending on your choice. For example:
"#[export] Hint Rewrite foo : bar."
[deprecated-hint-rewrite-without-locality,deprecated]
```

**Comment by khieta** [comment]
That's a warning from v8.14. It can be fixed similarly to the 8.13 warnings, but breaks compatibility with v8.12-8.13 :( So I've decided to ignore it for the time being.

**Comment by k4rtik** [comment]
oh, that makes sense, perfect!
